src/constants/earthEngine.js

```diff
@@ -254,7 +268,14 @@ export const earthEngineLayers = () => [
         params: {
             min: 0,
             max: 1500,
-            palette: '#ffffd4,#fee391,#fec44f,#fe9929,#d95f0e,#993404', // YlOrBr
+            palette: [
+                '#ffffd4',
+                '#fee391',
+                '#fec44f',
+                '#fe9929',
+                '#d95f0e',
+                '#993404',
+            ], // YlOrBr
```

Comment:
Is there a color that neatly describes this palette? (like Reds, Blues as the others have)